### ISSUE TITLE
Cannot connect to Billing Service

### ISSUE THREAD

**Issue by amarksdev** [code-with-max/godot-google-play-iapp#13]
Hello, I have some issues with connecting to the Billing API. The singleton gets loaded, it tries to connect but then it says something about an API version 3 and I don't know where this is coming from. Should be version 7, I guess. I mean, that's the goal of all this.

My log:
I/godot   (26634): AndroidIAPP singleton loaded
I/godot   (26634): Billing: start connection
W/BillingClient(26634): In-app billing API version 3 is not supported on this device.
W/BillingClient(26634): Billing service disconnected.
I/godot   (26634): Billing disconnected

Do I miss some setup? Do I have to include some additional plugins or dependencies in my build.gradle or permissions in AndroidManifest?

Thanks for any help.

**Comment by code-with-max** [comment]
This is very strange, I just made sure that I am using version 7.
No additional settings are needed, just provide the application with Internet access to connect to the server.
Try updating the plugin files from the releases section.

![2024-08-30 14-39-40](https://github.com/user-attachments/assets/5870262e-04a1-485f-b32e-7575ee8247ac)

**Comment by amarksdev** [comment]
Hi, thanks for your prompt reply. I used the files of the release V1.2 for Godot 4.2 as I was using Godot 4.2. Now, I tried to merge to Godot 4.3, deleted the android/build folder, reinstalled the according Android Build Template and then downloaded the files of the release V1.2 for Godot 4.3. That seems to work fine. There might have been some settings from an earlier approach, I tried out some other plugins before. Anyway, thanks for your work!
